code/modules/admin/db_ban/functions.dm

```diff
@@ -500,8 +581,13 @@
 						bantypesearch += "'PERMABAN' "
 
 
-			var/DBQuery/select_query = GLOB.dbcon.NewQuery("SELECT id, bantime, bantype, reason, job, duration, expiration_time, ckey, a_ckey, unbanned, unbanned_ckey, unbanned_datetime, edits, ip, computerid FROM [format_table_name("ban")] WHERE 1 [playersearch] [adminsearch] [ipsearch] [cidsearch] [bantypesearch] ORDER BY bantime DESC LIMIT 100")
-			select_query.Execute()
+			var/datum/db_query/select_query = SSdbcore.NewQuery({"
+				SELECT id, bantime, bantype, reason, job, duration, expiration_time, ckey, a_ckey, unbanned, unbanned_ckey, unbanned_datetime, edits, ip, computerid
+				FROM [format_table_name("ban")] WHERE 1 [playersearch] [adminsearch] [ipsearch] [cidsearch] [bantypesearch] ORDER BY bantime DESC LIMIT 100"}, sql_params)
```

Comment:
`WHERE 1`? I'm not sure I get why the 1 is there

Reply:
Because the queries insert `AND` at the start of each of them themselves, so this avoids invalid SQL statements.